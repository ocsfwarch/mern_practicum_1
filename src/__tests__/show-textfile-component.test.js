import React from 'react';
import TextFileList from '../components/show-textfilelist.component.js';
import renderer from 'react-test-renderer';

it('Renders a snapshot', () => {
  const match = {
    params:{email:"tpartcal@gmail.com"}
  };

  const tree = renderer.create(<TextFileList match={match}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
