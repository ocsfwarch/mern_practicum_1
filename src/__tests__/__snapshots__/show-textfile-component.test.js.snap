// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a snapshot 1`] = `
<div>
  <h3>
    Audio to Text File List
  </h3>
  <table
    className="table table-striped"
    style={
      Object {
        "marginTop": 20,
      }
    }
  >
    <thead>
      <th>
        Name
      </th>
      <th>
        Description
      </th>
      <th>
        EMail
      </th>
    </thead>
    <tbody />
  </table>
</div>
`;
